heliopy/data/util.py

```diff
@@ -403,6 +403,7 @@ def cdf2df(cdf, index_key, dtimeindex=True, badvalues=None):
     df : :class:`pandas.DataFrame`
         Data frame with read in data.
     """
+    ignore_key = ['Time3_PB5', 'Time_PB5']
```

Comment:
Can you add `ignore_key` as a keyword argument to `cdf2df`, and then put specific ignore keys in the individual data methods? That way it's easy to see in the methods which columns are being dropped.

Reply:
Sounds good! On it!